server/models/Profile.js

```diff
@@ -0,0 +1,51 @@
+const mongoose = require("mongoose");
+const addressSchema = require('./Address');
+const availabilitySchema = require('./Availability');
+const paymentCardSchema = require('./paymentCard');
+
+const STORAGE = 'https://placeholder.com/bucket'; //host for profile images
+
+const profileSchema = new mongoose.Schema({
+    _id: mongoose.Schema.ObjectId,
+    firstName: {type: String, required: true, default: ''},
+    lastName: {type: String, required: true, default: ''},
+    email: { type: String, required: true, default: ''},
+    description: {type: String, required: false, default: ''},
+    gender: {type: String, required: true},
+    address: addressSchema,
+    availability: [availabilitySchema],
+    payments: [paymentCardSchema],
+    profilePicture: {
+        type: String,
+        required: false,
+        get: p => `${STORAGE}${p}`
+    },
+    wage: {
+        type: Number,
+        min: [0, 'Number must be positive, got {VALUE}'],
+        get: w => `${w}/hr`
+    },
```

Comment:
For our payment provider Stripe, this might not be the best solution here. I would rather keep it as a number and do formatting on the client side.